business_requirement/README.rst

```diff
@@ -0,0 +1,363 @@
+.. |badge1| image:: https://img.shields.io/badge/maturity-Stable-green.png
```

Comment:
This should be built by the README generator. You need to fragment the README in pieces similar to https://github.com/OCA/mis-builder/tree/10.0/mis_builder/readme

Reply:
please review again, is it OK?